src/test/resources/config/application.yml

```diff
@@ -12,7 +12,16 @@
 # Full reference is available at:
 # http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
 # ===================================================================
-
+org:
```

Comment:
this is the most important part, which is missing in the main generator-jhipster, that's why the integration tests can't start